python/lsst/cp/pipe/cpCombine.py

```diff
@@ -201,9 +201,9 @@ class CalibCombineConfig(pipeBase.PipelineTaskConfig,
         doc="Mask planes to respect",
     )
     combine = pexConfig.Field(
-        dtype=int,
-        default=int(afwMath.MEANCLIP),
-        doc="Statistic to use for combination (from lsst.afw.math)",
+        dtype=str,
+        default='MEANCLIP',
+        doc="Statistic name to use for combination (from lsst.afw.math)",
```

Comment:
I guess the only option for the user to know the possible options is to look at `Statistics.h`, no?

Reply:
I think if this gets changed more than just to compare against eotest, then it probably should be made into a ChoiceField.